client/components/review.js

```diff
@@ -125,10 +126,33 @@ class Review extends React.Component {
     );
   }
 
+  changed = () => {
+    if (this.props.changedFromLatest) {
+      return LATEST;
+    }
+    else if (this.props.changedFromGranted) {
+      return GRANTED;
+    }
+  }
+
+  changedBadge = () => {
+    switch (this.changed()) {
+      case LATEST:
+        return <span className="badge changed">changed</span>;
+      case GRANTED:
+        return <span className="badge">amended</span>;
+      default:
+        return null;
+    }
```

Comment:
There's a lot of misdirection in this code now, calling out to another function and importing constants, to just do a simple condition on a boolean prop. 

I think we can reduce this function to:

```
changedBadge = () => {
  if (this.props.changedFromLatest) {
    return <span className="badge changed">changed</span>;
  }
  if (this.props.changedFromGranted) {
    return <span className="badge">amended</span>;  
  }
  return null;
}
```

And then get rid of `this.changed()` and the constants.